composer.json

```diff
@@ -10,6 +10,7 @@
         }
     ],
     "require": {
+        "chi-teck/drupal-code-generator": "4.x-dev#e61b917 as 3.5.0",
```

Comment:
This package was updated to Symfony 7, but there is no release.

Considering that the whole 11.x branch is "early preview", I think this should be okay to have like this.

Once Drupal 11 is in beta and all these packages have their releases, we can clean it up.

Reply:
I don't think you need to add `as 3.5.0` unless something is explicity needing that version. If you are using `composer require` it checks stability. But if you just add it manually to the composer.json it works fine. This is a feature afaik.

I updated it locally to be this and ran composer update. 
```
 "chi-teck/drupal-code-generator": "4.x-dev",
```

If you intention was to pin it then this is fine `4.x-dev#e61b917`. But what if they rewrite history in the dev branch, then i think this project would stop building (maybe that's a good thing).

Also, I like setting the default minimum stability to be alpha instead of dev. That seems pretty good for users.

Reply:
it worked for you because you probably did not use the `dev` version of core.

Drush 13.x requires `4.x-dev` for Symfony 7 compatibility https://github.com/drush-ops/drush/blob/13.x/composer.json#L36C3-L36C57

Reply:
`chi-teck/drupal-code-generator` is now released. waiting for Drush to start using it at `alpha+` version level.

Reply:
https://github.com/drush-ops/drush/pull/6010

Reply:
> it worked for you because you probably did not use the dev version of core.

Sorry I isn't sinking in why this project needs to use the dev version of core.